lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -69,6 +69,10 @@ sub _add_default_response {
   my ($self, $op_spec) = @_;
   my $name        = $self->{default_response_name};
   my $schema_data = $self->validator->schema->data;
+  my $codes       = $self->{default_response_codes};
+
+  # turn off with config { default_response_codes => [] }
+  return unless @$codes;
```

Comment:
Could you do `return unless @{$self->{default_response_codes}};` instead for less lines of code?

Reply:
done.